tests/jit/jit_models_test.py

```diff
@@ -106,7 +107,7 @@ def small_model_params():
     return params
 
 
-@pytest.mark.parametrize("model_def", [DCCRNet, DeMask])
+@pytest.mark.parametrize("model_def", [DPRNNTasNet])
```

Comment:
That doesn't look right. Why did you do that?

Reply:
I think we just add whatever model we just want to test with jit here, no?

Reply:
We need to keep all the tests, otherwise we don't know when we break things.

Reply:
I added them back